src/main/java/com/cloudbees/jenkins/plugins/amazonecr/AmazonECSRegistryCredential.java

```diff
@@ -60,30 +61,40 @@ public class AmazonECSRegistryCredential extends BaseStandardCredentials
 
     private final String credentialsId;
 
-    private final Regions region;
+    private final String region;
```

Comment:
Spotbugs complained about "non-transient non-serializable instance field region" when using the new `Region` class here, so I just used a String...